### DISCUSSION TITLE
请求增加更多的歌词字体大小

### DISCUSSION THREAD

**Discussion by Sakurashiyo** [SPlayer-Dev/SPlayer#184] [想法 Ideas]
因为有的时候听日语的歌的时候会出现发音，但是在这样的情况下哪怕是设置中最小的字体
依旧是有点太大了的感觉，所以请求增加更多的歌词字体大小...
版本：
使用的是Releases的2.0.9

**Comment by imsyy** [comment]
未来会添加用户自定义大小
